doc/man7/EVP_KDF-HKDF.pod

```diff
@@ -17,11 +17,38 @@ of the KDF).
 
 The output is considered to be keying material.
 
+=head2 Fixed-Digest HKDF
+
+B<HKDF-SHA256>, B<HKDF-SHA384> and B<HKDF-SHA512> are fixed-digest versions
+of the B<HKDF> algorithm. Each algorithm has its own OID. These algorithms
+are instantiated with the appropriate digest already configured, thus it is
+not necessary to set the digest using the B<OSSL_KDF_PARAM_DIGEST> parameter.
+An attempt to set the digest to anything other than the pre-configured digest
+will result in an error. L<EVP_KDF_CTX_reset(3)> will not reset the context's
+digest.
+
 =head2 Identity
 
-"HKDF" is the name for this implementation; it
+The following algorithms are available for this implementation; they
 can be used with the EVP_KDF_fetch() function.
 
+In this list, names are grouped together to signify that they are the same
+algorithm having multiple names.  This also includes the OID in canonical
+decimal form (which means that they are possible to fetch if the caller has a
+mere OID which came out in this form after a call to L<OBJ_obj2txt(3)>).
+
+=over 4
+
+=item "HKDF"
+
+=item "HKDF-SHA256", "id-alg-hkdf-with-sha256", "1.2.840.113549.1.9.16.3.28"
+
+=item "HKDF-SHA384", "id-alg-hkdf-with-sha384", "1.2.840.113549.1.9.16.3.29"
+
+=item "HKDF-SHA512", "id-alg-hkdf-with-sha512", "1.2.840.113549.1.9.16.3.30"
+
+=back
+
```

Comment:
I would probably update the example below also to use the new form..

Reply:
@slontis, looking for clarification for your comment above.

Reply:
@slontis is on vacation at the moment.